Automattic/wp-calypso
# Stats: remove mixin-skeleton and mixin-toggle usage.

With the merge of #2863 there is no more usage of the old stats mixins that supported the "toggle" module feature and the skeleton/loading data feature.

This branch removes all existing references to `mixin-skeleton` and removes the file itself.  `mixin-toggle` still is used on the post detail page - which still also needs to be updated to utilize the standard section headers.  As such, this branch does not remove `mixin-toggle` which is powering the (i) info bubble toggles.  I will tackle that update in a subsequent PR.

__To Test__
-  Open a site stats page for a time period ( day / week etc ), verify all looks happy, specifically loading indicators still work.
- Likewise, open an insights page and ensure loading indicators still work
